Makefile

```diff
@@ -45,34 +45,15 @@ run-cleanup-with-update: | setup-docker-compose
 	docker-compose -f dc.test.yml exec $(MODULE) ./cleanup.sh
 
 # PROD
-
-# pass env variables to container
 build_args=--build-arg MODULE=$(MODULE) --build-arg PORT=$(PORT)
 
 build:
-	docker build -t openslides-datastore-$(MODULE) . $(build_args)
-
-run: | build
-	docker-compose up -d $(MODULE)
-
-run-verbose: | build
-	docker-compose up $(MODULE)
-
-
-build-prod:
-	docker build -t openslides-datastore-$(MODULE) -f Dockerfile.prod . $(build_args)
-
-run-prod: | build-prod
-	docker-compose -f dc.prod.yml up -d $(MODULE)
-
-run-prod-verbose: | build-prod
-	docker-compose -f dc.prod.yml up $(MODULE)
+	docker build -t openslides-datastore-$(MODULE) $(build_args) .
```

Comment:
Why do you need the `build.sh` if you don't use them here?

Reply:
They are used in the global setup: The build script there checks, if a build.sh exists in the repo (here reader/writer dir). If so it is executed, if not it is assumes that there is a Dockerfile. This works for all others, but since we have a unified one, we need the extra script. We cannot use the Makefile, since we need extra args (tag name, docker repository, ... ) for the build command and Makefiles are not the right tool for it.

Reply:
If it is ok, please merge

Reply:
ok